docs/Initial-Setup/Setting-up-your-Manager-Instance.rst

```diff
@@ -71,7 +71,7 @@ before, so we need to first ssh into the instance and make sure the
 setup is complete.
 
 In either case, ``ssh`` into your instance (e.g. ``ssh -i firesim.pem centos@YOUR_INSTANCE_IP``) and wait until the
-``~/machine-launchstatus`` file contains all the following text:
+``/machine-launchstatus`` file contains all the following text:
```

Comment:
Is the `machine-launchstatus` file at `/` now?

Reply:
Yes.  It's the fallout of just doing https://github.com/firesim/firesim/pull/932#discussion_r809629052

Reply:
Because it was previously hardcoded to put it in `/home/centos` and that won't exist when not on an Centos VM